08/calculator.go

```diff
@@ -0,0 +1,5 @@
+package calculator
+
+func Add(x int, y int) int {
```

Comment:
We can also write this as `x, y int` if we like (but for only two parameters, it looks just as good to me either way. If we had 10 integer parameters, we would probably use the shorter style. But if we had 10 parameters, there may be a problem with our design!)